crates/spirv-std/src/arch/ray_tracing.rs

```diff
@@ -0,0 +1,175 @@
+use crate::{ray_tracing::*, vector::Vector};
+
+/// Converts a 64-bit integer into an [`AccelerationStructureKHR`].
+/// # Safety
+/// The 64-bit integer must point to a valid acceleration structure.
+#[spirv_std_macros::gpu_only]
+#[doc(alias = "OpConvertUToAccelerationStructureKHR")]
+#[allow(clippy::empty_loop)]
+#[inline]
+pub unsafe fn convert_u_to_acceleration_structure_khr(id: u64) -> AccelerationStructureKhr {
+    // Since we can't represent an uninitalized opaque type in Rust at the
+    // moment, we need to create and return the acceleration structure entirely
+    // in assembly.
+    asm! {
+        "%ret = OpTypeAccelerationStructureKHR",
+        "%result = OpConvertUToAccelerationStructureKHR %ret {id}",
+        "OpReturnValue %result",
+        "%blah = OpLabel",
```

Comment:
Small nit :-)

Reply:
Nit that it's called blah or that it's there at all? The `OpLabel` has to be there because currently we don't correctly handle terminators in inline assembly. https://github.com/EmbarkStudios/rust-gpu/issues/564

Reply:
Yeah just about the name.